src/test/resources/test.properties

```diff
@@ -17,5 +16,7 @@ log.level=INFO
 #cache=false | true
 #demo.mode=false | true
 #demo.delay=2
-#driver.remote.url=http://localhost:4444/wd/hub
-#screenshot.strategy=on fail | on | off
\ No newline at end of file
+#screenshot.strategy=on fail | on | off
+
+#remote.type=sauce
```

Comment:
This option should be enabled. If it's not working properly on jdi level, please create an issue in jdi-light project to fix Sauce labs integration

Reply:
It's working properly on jdi level. I placed these properties in a separate group. The reason is that we need to remove comments about this at the same time while running the test suite in Sauce Labs. Also we need to replace <SAUCE_USERNAME> and <SAUCE_ACCESS_KEY> tags with the corresponding user account values.
#remote.type=sauce
#driver.remote.url=https://<SAUCE_USERNAME>:<SAUCE_ACCESS_KEY>@ondemand.us-west-1.saucelabs.com:443/wd/hub

Reply:
Removed comments and improved remote url:
remote.type=sauce
driver.remote.url=https:ondemand.us-west-1.saucelabs.com:443/wd/hub